1_html-css/CSS_CV.css

```diff
@@ -0,0 +1,150 @@
+body {
```

Comment:
The CSS is not very well formatted. Try to clean it up and/or search on the internet how to automate it with Prettier!

Reply:
Prettier is on on my CSS as well.
Also, one more section that was missing has been added.